extensions/wikia/PortableInfobox/styles/PortableInfobox.scss

```diff
@@ -81,6 +81,35 @@
 	text-align: center; // center smaller image inside infobox
 }
 
+.pi-image-gallery {
+	position: relative;
+
+	&::after {
+		background-image: url("/extensions/VisualEditor/modules/ve-mw/ui/styles/images/icons/gallery-ltr-white.svg");
+		background-repeat: no-repeat;
+		background-size: contain;
+		content: '';
+		display: block;
+		height: 30px;
+		position: absolute;
+		right: $infobox-item-margin;
+		top: $infobox-item-margin;
+		width: 30px;
+	}
+}
+
+.pi-gallery-caption {
+	color: white;
```

Comment:
color declarations should be stored in sass variables. Please run sass-linter for this file to check for other issues